paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -142,12 +144,36 @@ internal class CustomerSheetViewModel @Inject constructor(
             is CustomerSheetViewAction.OnUpdateMandateText -> {
                 updateMandateText(viewAction.mandateText, viewAction.showAbovePrimaryButton)
             }
-            is CustomerSheetViewAction.OnUSBankAccountRetrieved -> {
-                onUSBankAccountRetrieved(viewAction.usBankAccount)
+            is CustomerSheetViewAction.OnCollectBankAccountResult -> {
+                onCollectUSBankAccountResult(viewAction.bankAccountResult)
+            }
+            is CustomerSheetViewAction.OnConfirmUSBankAccount -> {
+                onConfirmUSBankAccount(viewAction.usBankAccount)
             }
             is CustomerSheetViewAction.OnFormError -> {
                 onFormError(viewAction.error)
             }
+            is CustomerSheetViewAction.OnCancelClose -> {
+                onCancelCloseForm()
+            }
+        }
+    }
+
+    // If true, the bottom sheet will be dismissed, otherwise the sheet will stay open
```

Comment:
Nit to make it show up in docs.

```suggestion
    /**
     * If true, the bottom sheet will be dismissed, otherwise the sheet will stay open
     */
```